roles/install-etcd/tasks/main.yml

```diff
@@ -0,0 +1,36 @@
+- name: Download and setup etcd
+  unarchive:
+    src: "https://storage.googleapis.com/etcd/{{ etcd_version }}/etcd-{{ etcd_version }}-linux-ppc64le.tar.gz"
```

Comment:
see if you can use the arch `ppc64le` from some ansible env to avoid hardcoded value